cs/cli/vw_base.cpp

```diff
@@ -237,9 +237,12 @@ void VowpalWabbitBase::SaveModel(String^ filename)
     throw gcnew ArgumentException("Filename must not be null or empty");
 
   String^ directoryName = System::IO::Path::GetDirectoryName(filename);
-
   if (!String::IsNullOrEmpty(directoryName))
-  { System::IO::Directory::CreateDirectory(directoryName);
```

Comment:
whats wrong with this api? is it different than the one you mention CreateDirectoryA?

Reply:
Some header change I think caused it to not be able to find this function any more. I figured since the .net core change is incoming I would not spend too much time on it